mws/mws.py

```diff
@@ -11,21 +11,21 @@
 import hmac
 import re
 import warnings
-from zipfile import ZipFile
-from io import BytesIO
 
-from requests import request
+import chardet
```

Comment:
Is this library necessary for it to be a dependancy we need to watch. I know in the past we have been quite against adding other libraries as its another thing to manage / deal with version changes etc.

Reply:
It's an indirect dependency through requests we already now have to watch, i should add that as a comment.
https://github.com/psf/requests/blob/eedd67462819f8dbf8c1c32e77f9070606605231/setup.py#L45
I made it a direct dependency now too. I agree this part in general is opionated, it just works to 99% instead of 90% (made up numbers, in particular considerable for europe).
I was doing some deep digging in this part, i even tried to make a pull request for chardet.
It's a bug especially for processing non literature european text (sample to identify encoding is low). He commented on my PR: https://github.com/kakulukia/pypugjs/issues/56 .
I can write an argumentation for the alternative options.